tests/cwl/log-messages.feature

```diff
@@ -0,0 +1,16 @@
+# Tests for the expected log messages
```

Comment:
Cosmetics: the commit log message says:

```
Changes the REANA specification files so that the output file is always
created in the same directory, regardless of the workflow engine.
```

This part should be probably removed.